src/Math-Matrix/PMSingularValueDecomposition.class.st

```diff
@@ -50,7 +46,7 @@ PMSingularValueDecomposition class >> decompose: aMatrix [
 
 { #category : #initialization }
 PMSingularValueDecomposition >> initialize: aMatrix [
-	| symU symV eigenU eigenV diag |
+	| symU symV eigenU eigenV diag m n |
```

Comment:
Maybe move these so that the order follows the actual assignments

Reply:
ok done